src/main/java/no/digipost/security/cert/ProdEnvCertificates.java

```diff
@@ -0,0 +1,175 @@
+        return BuypassClass3CaG2HT.cert;
+    }
+
+
+    /**
+     * Buypass Root CA (SEID generation 1)
+     *
+     * <p>CN=Buypass Class 3 Root CA, O=Buypass AS-983163327, C=NO
+     * <p>valid 2010-10-26 to 2040-10-26
+     */
+    public static X509Certificate buypassClass3RootCa() {
+        return BuypassClass3RootCa.cert;
+    }
+
+    /**
+     * Buypass certificate issuer CA (SEID generation 1)
+     *
+     * <p>CN=Buypass Class 3 CA 3, O=Buypass AS-983163327, C=NO
+     * <p>valid 2012-09-25 to 2032-09-25
+     */
+    public static X509Certificate buypassClass3Ca3() {
+        return BuypassClass3Ca3.cert;
+    }
+
+    /**
+     * Commfides Root CA (SEID generaation 1)
+     *
+     * <p>C=NO, O=Commfides Norge AS - 988 312 495, OU=Commfides Trust Environment (c) 2011 Commfides Norge AS, CN=CPN RootCA SHA256 Class 3
+     * <p>valid 2011-02-28 to 2025-01-01
+     */
+    public static X509Certificate commfidesRootCa() {
+        return CommfidesRootCa.cert;
+    }
+
+    /**
+     * Commfides certificate issuer CA (SEID generation 1)
+     *
+     * <p>C=NO, O=Commfides Norge AS - 988 312 495, OU=Commfides Trust Environment (c) 2011 Commfides Norge AS, CN=CPN Enterprise SHA256 CLASS 3
+     * <p>valid 2011-02-28 to 2024-12-31
+     */
+    public static X509Certificate commfidesCa() {
+        return CommfidesCa.cert;
+    }
+
+    /**
+     * Digipost Root CA
+     *
+     * <p>CN=Digipost Root CA, OU=Digipost, O=Posten Norge AS, ST=Oslo, C=NO
+     * <p>valid 2017-10-06 to 2050-01-01
+     */
+    public static X509Certificate digipostRootCa() {
+        return DigipostRootCa.cert;
+    }
+
+
+    // X509Certificate singletons
+
+    private static final class BuypassClass3RootCa {
+        static final X509Certificate cert = readCertificate("BPClass3RootCA.cer");
+    }
+
+    private static final class BuypassClass3Ca3 {
+        static final X509Certificate cert = readCertificate("BPClass3CA3.cer");
+    }
+
+    private static final class BuypassClass3RootCaG2ST {
+        static final X509Certificate cert = readCertificate("BPCl3RootCaG2ST.cer");
+    }
+
+    private static final class BuypassClass3CaG2ST {
+        static final X509Certificate cert = readCertificate("BPCl3CaG2STBS.cer");
+    }
+
+    private static final class BuypassClass3RootCaG2HT {
+        static final X509Certificate cert = readCertificate("BPCl3RootCaG2HT.cer");
+    }
+
+    private static final class BuypassClass3CaG2HT {
+        static final X509Certificate cert = readCertificate("BPCl3CaG2HTBS.cer");
+    }
+
+
+    private static final class CommfidesRootCa {
+        static final X509Certificate cert = readCertificate("commfides_root_ca.cer");
+    }
+
+    private static final class CommfidesCa {
+        static final X509Certificate cert = readCertificate("commfides_ca.cer");
+    }
+
+
+    private static final class DigipostRootCa {
+        static final X509Certificate cert = readCertificate("digipost_root_ca.cert.pem");
+    }
```

Comment:
Hva er det vi bruker denne til, er det sånn att vi kan bruke self-signed sert for testing?